src/v3/src/components/ImageLink/ImageLink.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box } from '@mui/material';
+import { Link } from '@okta/odyssey-react-mui';
+import { ImageLinkElement, UISchemaElementComponent } from 'src/types';
+
+const ImageLink: UISchemaElementComponent<{
+  uischema: ImageLinkElement
+}> = ({ uischema }) => {
+  const { options } = uischema;
+  const Icon = options.svgIcon;
+  return (
+    <Box>
+      <Link
+        textAlign="center"
+        href={options.url}
+        rel="noopener noreferrer"
+        aria-label={options.altText}
```

Comment:
```suggestion
        ariaLabel={options.altText}
```

Reply:
Btw, SIW gen3 now has a mix of MUI components imported straight from `@mui/material` and some from `@okta/odyssey-react-mui`. In this case, `Link` is from Odyssey and has a more restrictive set of props that it can take in, so it doesn't allow lots of the global HTML attributes like `aria-X` out of the box. You have to click into the component's source within Odyssey to see what props it actually takes